### PR TITLE
test: add tests for label and config file in daemon

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

### Ⅰ. Describe what this PR did
Add deamon tests for ：
1. label
2. configure file

### CODE REVIEW

File: test/daemon/daemon.go [hunk 1]
```diff
@@ -148,7 +152,17 @@ func (d *Config) StartDaemon() error {
 	if util.WaitTimeout(time.Duration(d.timeout)*time.Second, d.IsDaemonUp) == false {
 		if d.Debug == true {
 			d.DumpLog()
+
+			fmt.Printf("\n")
 			fmt.Printf("Failed to launch pouchd:%v\n", d.Args)
```
Comment (HusterWan) [inline test/daemon/daemon.go]: maybe we could use `fmt.Printf("\nFailed to launch pouchd:%v\n", d.Args)`

File: test/z_cli_daemon_test.go [hunk 2]
```diff
@@ -72,26 +76,115 @@ func (suite *PouchDaemonSuite) TestDaemonCgroupParent(c *check.C) {
...
+		dcfg.Listen = ""
+		dcfg.NewArgs("--listen=" + addr)
+		err := dcfg.StartDaemon()
 		c.Assert(err, check.IsNil)
 
 		// verify listen to tcp works
 		command.PouchRun("--host", addr, "version").Assert(c, icmd.Success)
 
 		dcfg.KillDaemon()
+		defer dcfg.KillDaemon()
```
Comment (HusterWan) [inline test/z_cli_daemon_test.go]: why KillDaemon() twice?

File: test/z_cli_daemon_test.go [hunk 3]
```diff
@@ -72,26 +76,115 @@ func (suite *PouchDaemonSuite) TestDaemonCgroupParent(c *check.C) {
...
+	err := CreateConfigFile(path, cfg)
+	c.Assert(err, check.IsNil)
+	defer os.Remove(path)
+
+	dcfg, err := StartDefaultDaemon("--containerd-path", "def", "--config-file="+path)
+	dcfg.KillDaemon()
+	c.Assert(err, check.NotNil)
+}
+
+// TestDaemonConfigFileUnknownFlag tests start daemon with unknow flags in configure file.
```
Comment (HusterWan) [inline test/z_cli_daemon_test.go]: %s/unknow/unknown

File: test/daemon/daemon.go (hunk 1 above)
Comment (Letty5411) [reply test/daemon/daemon.go]: good

**Comment by Letty5411** [comment]
@HusterWan code is ready, PTAL.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**